CONTRIBUTING.md

```diff
@@ -152,8 +152,17 @@ docs: fix typos
 
 #### Reviews & Approvals
 
-All pull requests must be reviewed and approved by at least one Vector team
-member. The review process is outlined in the [Review guide](REVIEWING.md).
+All pull requests should be reviewed by:
+
+- No review required for cosmetic changes like whitespace, typos, and spelling
```

Comment:
This is technically not correct since we do need to review contributor trivial changes.

Reply:
:+1: 10e50b14